From 28434e4b9068e679089a416cd267a8e09a2a5432 Mon Sep 17 00:00:00 2001
From: student <student@ReliableEmbeddedSystems.com>
Date: Wed, 21 Dec 2022 23:44:18 +0000
Subject: [PATCH] added langdale support

Signed-off-by: student <student@ReliableEmbeddedSystems.com>
---
 conf/layer.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/layer.conf b/conf/layer.conf
index 9d0f6f3..ed3403d 100644
--- a/conf/layer.conf
+++ b/conf/layer.conf
@@ -11,7 +11,7 @@ BBFILE_PRIORITY_swupdate-boards = "7"
 
 LAYERDEPENDS_swupdate-boards = "swupdate"
 
-LAYERSERIES_COMPAT_swupdate-boards = "kirkstone"
+LAYERSERIES_COMPAT_swupdate-boards = "kirkstone langdale"
 
 BBFILES_DYNAMIC += " \
     atmel:${LAYERDIR}/dynamic-layers/atmel/*/*/*.bb \
-- 
2.34.1

