From d1cc8b27cbd4fd6dd1e2a0dce12faf610adfd6c6 Mon Sep 17 00:00:00 2001
From: student <student@ReliableEmbeddedSystems.com>
Date: Tue, 18 Jun 2024 13:32:42 +0000
Subject: [PATCH] upgrade to scarthgap styhead

Signed-off-by: student <student@ReliableEmbeddedSystems.com>
---
 conf/layer.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf/layer.conf b/conf/layer.conf
index 9d0f6f3..5034abf 100644
--- a/conf/layer.conf
+++ b/conf/layer.conf
@@ -11,7 +11,7 @@ BBFILE_PRIORITY_swupdate-boards = "7"
 
 LAYERDEPENDS_swupdate-boards = "swupdate"
 
-LAYERSERIES_COMPAT_swupdate-boards = "kirkstone"
+LAYERSERIES_COMPAT_swupdate-boards = "kirkstone scarthgap styhead"
 
 BBFILES_DYNAMIC += " \
     atmel:${LAYERDIR}/dynamic-layers/atmel/*/*/*.bb \
-- 
2.34.1

