From 3bef698ee85d381a9a107fb2945a761348442e49 Mon Sep 17 00:00:00 2001
From: Robert Berger <Robert.Berger@ReliableEmbeddedSystems.com>
Date: Mon, 30 Jun 2025 20:59:44 +0000
Subject: [PATCH 28/31] glome: new syntax

Signed-off-by: Robert Berger <Robert.Berger@ReliableEmbeddedSystems.com>
---
 recipes-security/glome/glome_git.bb | 1 -
 1 file changed, 1 deletion(-)

diff --git a/recipes-security/glome/glome_git.bb b/recipes-security/glome/glome_git.bb
index b99239e..5a0300f 100644
--- a/recipes-security/glome/glome_git.bb
+++ b/recipes-security/glome/glome_git.bb
@@ -10,7 +10,6 @@ inherit meson pkgconfig
 
 DEPENDS += "openssl"
 
-S = "${UNPACKDIR}/git"
 SRC_URI = "git://github.com/google/glome.git;branch=master;protocol=https"
 SRCREV = "48d28f82bd51ae4bccc84fbbee93c375b026596b"
 
-- 
2.34.1

