From 13a28f00a5dbabfda9a3da7e58c76ba8091dff13 Mon Sep 17 00:00:00 2001
From: Robert Berger <Robert.Berger@ReliableEmbeddedSystems.com>
Date: Fri, 6 Sep 2024 19:11:47 +0000
Subject: [PATCH] go-vendor: WORKDIR -> UNPACKDIR

Signed-off-by: Robert Berger <Robert.Berger@ReliableEmbeddedSystems.com>
---
 meta/classes/go-vendor.bbclass | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta/classes/go-vendor.bbclass b/meta/classes/go-vendor.bbclass
index 6f43a09a40..12667b2d60 100644
--- a/meta/classes/go-vendor.bbclass
+++ b/meta/classes/go-vendor.bbclass
@@ -156,7 +156,7 @@ python do_go_vendor() {
                 shutil.copy2(rootdirLicese, subdirLicense)
 
     # Copy vendor manifest
-    modules_txt_src = os.path.join(d.getVar('WORKDIR'), "modules.txt")
+    modules_txt_src = os.path.join(d.getVar('UNPACKDIR'), "modules.txt")
     bb.debug(1, "cp %s --> %s" % (modules_txt_src, vendor_dir))
     shutil.copy2(modules_txt_src, vendor_dir)
 
-- 
2.34.1

